ihatemoney/templates/layout.html

```diff
@@ -144,14 +145,14 @@ <h6 class="dropdown-header">{{ _('Languages') }}</h6>
             <a target="_blank" rel="noopener" data-toggle="tooltip" data-placement="top" title="{{ _('Code') }}" href="https://github.com/spiral-project/ihatemoney">
               <i class="icon git">{{ static_include("images/git.svg") | safe }}</i>
             </a>
-            <a target="_blank" rel="noopener" data-toggle="tooltip" data-placement="top" title="{{ _('Mobile Application') }}" href="https://gitlab.com/eneiluj/moneybuster#moneybuster-for-android">
+            <a target="_blank" rel="noopener" data-toggle="tooltip" data-placement="top" title="{{ _('Mobile Application') }}" href="{{url_for('main.downloadMobileApp')}}">
```

Comment:
```suggestion
            <a data-toggle="tooltip" data-placement="top" title="{{ _('Mobile Application') }}" href="{{url_for('main.downloadMobileApp')}}">
```
Remove attributes needed when opening a new tab. Now, this is an internal link.